nix/overlay.nix

```
@@ -59,6 +59,23 @@ self: super: {
     cp $src/terraform-provider-hcloud $out/bin
   '';
 
+  nix = super.nix.overrideAttrs (drv: {
+    patches = drv.patches or [] ++ [
+      # Extra patch to catch more evaluation errors, remove when included in nix stable.
+      # https://github.com/NixOS/nix/pull/3584
+      (self.fetchpatch {
+        name = "outputs-to-install-validate.patch";
+        url = "https://github.com/NixOS/nix/pull/3584/commits/e022ce892e35c49e6091c8baa92881ba4544ff65.patch";
```

It redirects to this URL, anyways (unless this was intentional).

```suggestion
        url = "https://github.com/NixOS/nix/commit/e022ce892e35c49e6091c8baa92881ba4544ff65.patch";
```